### DISCUSSION TITLE
Using Pyshark with Wireshark Portable?

### DISCUSSION THREAD

**Discussion by flancast90** [KimiNewt/pyshark#533] [Q&A]
The name says it all, I would like to point PyShark to my portable installation of Wireshark. I have read about changing the .ini file, but not sure if this would work since the portable version is different than the regular...

Thanks for the help!

**Comment by KimiNewt** [comment]
I'm not familiar with the portable version, but it seems like only contains a wireshark executable. If there's no way to run tshark in that mode, then pyshark can't run. 

Running the executable with `--help` might tell us if there's a way to extract tshark from it
